requirements.txt

```diff
@@ -0,0 +1 @@
+pyvista==0.33.2
```

Comment:
Eh... I'm not a fan of this being here. Because we could easily get mismatched versions of the Docker image and PyVista. IMO, we should not include pyvista in the `requirements.txt` or leave the version unpinned as the version will be controlled by the Docker image

Reply:
```suggestion
pyvista
```